tests/integration/sriov_test.py

```diff
@@ -30,6 +32,11 @@
 SRIOV_CONFIG = {Ethernet.SRIOV_SUBTREE: {Ethernet.SRIOV.TOTAL_VFS: 2}}
 
 
+pytestmark = pytest.mark.skipif(
+    os.environ.get("CI") == "true", reason="CI devices do not support SR-IOV",
```

Comment:
How about using `not os.path.exists("/sys/class/net/eth1/device/sriov_numvfs")` as a condition? Ideally `eth1` would be a constant here, though.

Reply:
The thing is that we do not expect to have eth1 with SR-IOV support in CI soon. So I prefer to use the existing CI environment variable. If you are not fine with this I can change it, it is not a must for me :-)

Reply:
By actually checking the support on eth1, it would also skip the tests when running locally on a system without SR-IOV support. Actually, it would also allow to remove all the xfails. Also the CI variable has already too many implications (for example it also controls whether the working copy is copied in the container and whether to rebuild the container images), therefore I am not convinced that using it here, too, is a good idea.